### ISSUE TITLE
Mobile wizard layout (all steps) on new bug form

### ISSUE THREAD

**Issue by munteanu210** [webcompat/webcompat.com#2982]
Implement the wizard layout and content for the main categories, all sub-categories, browser & device selection, following the [mobile UI/UX](https://miro.com/app/board/o9J_kxX9vL0=/) on the new bug form.

**Comment by karlcow** [comment]
Does that mean there are two different versions for desktop and mobile? Instead of one responsive version?

**Comment by miketaylr** [comment]
@munteanu210 ^^

**Comment by munteanu210** [comment]
> Does that mean there are two different versions for desktop and mobile? Instead of one responsive version?

@karlcow, there is a single version. The mobile view is all about style updates that make the page responsive.

**Comment by munteanu210** [comment]
I fixed this issue in [this commit](https://github.com/webcompat/webcompat.com/pull/2984/commits/6ca85625dbc2c10e1add86382d15f167ddc3161a) which is a part of PR #2938. The commit may seem large, but the updates are minor. For example, it adds an HTML class so that it has specific behavior on mobile. Also, I changed the CSS to be mobile-first as seen on the project. This being said, a large block of desktop overrides was added.
